src/Controller/AdminController.php

```diff
@@ -887,11 +908,12 @@ public function taxationSettingsAction(
     }
 
     /**
-     * @Route("/admin/settings/tags", name="admin_tags")
+     * @Route("/admin/tags", name="admin_tags")
```

Comment:
URLs for time slots & packages also have the `/settings` sub-path, don't they need to be changed too? Or it's only needed for tags, as dispatchers are allowed to manipulate tags.

Reply:
Indeed, the change has been made for the `tags` because I have passed over them when creating the role dispatcher. But for the sake of consistency, it would be nice to change the url for the time slots and packages. For me the `/settings` route should contain the "technical" settings of the instance and not the CRUD of the business entities